src/modules/dashboard/components/CoinMachine/TokenSalesTable/TokenSalesTable.tsx

```diff
@@ -183,8 +183,8 @@ const TokenSalesTable = ({
   useEffect(() => {
     if (periodRemainingMS > 1000 && periodRemainingMS < periodLengthMS) {
       setTimeout(() => {
-        refetchSalePeriodsData(salePeriodQueryVariables);
         startPollingSalePeriodsData(periodLengthMS);
+        refetchSalePeriodsData(salePeriodQueryVariables);
```

Comment:
Don't we also need `salePeriodQueryVariables` as dependencies to the `useEffect` ?